app/lib/package.rb

```diff
@@ -161,4 +170,12 @@ def include_tips?
     language_attributes = @translation.language.language_attributes.where(resource: @translation.resource)
     language_attributes.find_by(key: "include_tips")&.value == "true"
   end
+
+  def add_file_to_zip_and_s3(zip, zip_filename, local_filename)
```

Comment:
Doesn't `sha_filename` remain a better name for the 2nd param?

Reply:
I mean the 2nd param, `zip_filename` - isn't it still a `sha_filename`?

Reply:
Oh.. for that one, same idea, there's no enforcing that (which is the path of the file inside the zip archive) is actually going to be sha, so I was making the name to reflect that it could be anything

Reply:
got it; sure!